Components/Container/index.js

```diff
@@ -9,6 +9,7 @@ export const Container = styled.section`
       flex-wrap: wrap;
       align-items: ${(props) => (props.$center ? "center" : "")};
       justify-content: space-around;
+      gap: ${(props) => (props.$gap ? "4px" : "")};
```

Comment:
I think i would explicitly set a value if no gap prop has been passed into the component.  I would also change the align-items above to have a default value of `start`

```suggestion
      gap: ${(props) => (props.$gap ? "4px" : "0")};
```

Reply:
Changed to 0px and start.